setup.py

```diff
@@ -110,8 +110,12 @@ def find_rel_test():
     provides=["eb"] + easybuild_packages,
     test_suite="test.framework.suite",
     zip_safe=False,
-    install_requires=["vsc-base >= 2.2.6"],
+    install_requires=[
+        'setuptools',
```

Comment:
the minimal required version is quite old, basically all we need is `pkg_resources.declare_namespace` and `pkg_resources.fixup_namespace_packages`; both are available in 0.6:

```
-bash-4.1$ python -c "import setuptools; print setuptools.__version__; from pkg_resources import declare_namespace, fixup_namespace_packages"
0.6
-bash-4.1$ 
```